tim1/src/main/java/isamrs/tim1/repository/RegisteredUserRepository.java

```diff
@@ -11,7 +11,7 @@
 
 	RegisteredUser findOneByEmail(String email);
 
-	@Query(value = "select * from users u where u.first_name like ?1 and u.last_name like ?2 and u.email not like ?3", nativeQuery = true)
+	@Query(value = "select * from users u where u.first_name like ?1 and u.last_name like ?2 and u.email not like ?3 and dtype like 'RegisteredUser'", nativeQuery = true)
```

Comment:
Query should be "select * from users u where lower(u.first_name) like ?1 and lower(u.last_name) like ?2 and u.email not like ?3 and dtype like 'RegisteredUser'"

Reply:
Suggestion is accepted, it will be corrected in next request